src/modules/extra/m_ssl_openssl.cpp

```diff
@@ -999,7 +1040,9 @@ class ModuleSSLOpenSSL : public Module
 	ModuleSSLOpenSSL()
 	{
 		// Initialize OpenSSL
-		OPENSSL_init_ssl(0, NULL);
+		if (!(OPENSSL_init_ssl(OPENSSL_INIT_LOAD_SSL_STRINGS, NULL))) {
```

Comment:
I think that `OPENSSL_INIT_LOAD_SSL_STRINGS` is the default if you pass no options (i.e. 0). Is this not correct?

Reply:
Looks like it currently is according to the online man page.